### PR TITLE
Add Exercise06

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise06/Program.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+namespace Exercise06
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var n = 100;
+
+            var ints = Enumerable.Range(2, n);
```
Comment (voquanghoa) [inline Exercise06/Program.cs]: `ints` ==> `numbers`
`list` ==> `primeNumbers`
`i` ==> `primeNumber`

Comment (huynhngocminh2511) [reply Exercise06/Program.cs]: Done

File: Exercise06/Program.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace Exercise06
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var n = 100;
+
+            var numbers = Enumerable.Range(2, n);
```
Comment (voquanghoa) [inline Exercise06/Program.cs]: `Enumerable.Range(2, 100);` ==> `2 3 4 .... 100 101` ==> wrong

File: Exercise06/Program.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            var primeNumbers = numbers.TakeWhile(x => x < n)
+                .Where(x => !numbers.TakeWhile(y => y <= (int)Math.Sqrt(x))
```
Comment (voquanghoa) [inline Exercise06/Program.cs]: Although the result is correct, the solution is still bad. Try to make it better. Bellow is my solution
```C#
            var n = 100;

            var numbers = Enumerable.Range(2, n-1).ToList();

            var primeNumbers = numbers
                .Where(x => Enumerable.Range(2, (x - 1)/2).All(y => x % y > 0))
                .ToList();

            primeNumbers.ForEach(Console.WriteLine);

            Console.ReadKey();
```

File: Exercise06/Program.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise06/Program.cs]: But TakeWhile(x => x < n) :v

Comment (voquanghoa) [reply Exercise06/Program.cs]: You're introducing a bug and then fix same time.  Sad

File: Exercise06/Program.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Exercise06/Program.cs]: Done

File: Exercise06/Program.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise06/Program.cs]: Don't sad. i had remove it :v

File: Exercise06/Program.cs [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var n = 100;
+
+            var numbers = Enumerable.Range(2, n - 1);
+
+            var primeNumbers = numbers
+                .Where(x => !Enumerable.Range(2, (x - 1) / 2).Any(y => x % y == 0))
```
Comment (voquanghoa) [inline Exercise06/Program.cs]: Replace `!...Any(...)` by `All(...)`

Comment (huynhngocminh2511) [reply Exercise06/Program.cs]: Done

**Pull request merged**